corehq/apps/email/static/email/js/email_settings.js

```diff
@@ -0,0 +1,30 @@
+hqDefine('email/js/email_settings', [
+    "jquery",
+], function ($) {
+    $(document).ready(function() {
+        var saveButton = $('button[type="submit"]');
```

Comment:
This should be implemented using knockoutjs instead of directly meddling with the attributes here.

Reply:
Using knockoutjs now.